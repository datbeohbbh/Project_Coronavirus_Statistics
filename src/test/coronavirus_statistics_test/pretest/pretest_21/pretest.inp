29 27
? 5
q 1 2 5
? 4 1
q 1 4 260
q 1 6 445
? 4 3
? 7
q 1 20 60
? 6
q 1 15 321
q 1 7 329
? 4 4
? 4 1
? 4 5
q 3 7 26
? 6
q 1 2 484
q 1 22 310
q 3 15 220
q 2 20 13
? 7
? 4 1
? 4 6
? 4 4
? 4 1
? 6
? 4 4