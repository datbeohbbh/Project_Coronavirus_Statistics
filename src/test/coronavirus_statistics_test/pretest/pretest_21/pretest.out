Query ? 5 :
The number of safe cities is 29

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 5
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 445
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 4 :
The answer is the city with the index 15 
- Information of the city 15 -
-> Number of current patients : 321
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 5
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 329
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 7 :
The city with the highest death is city with the index 15
Quantity: 220 people

Query ? 4 1 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 47
-> Has recovered : 13
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 445
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 303
-> Has recovered : 0
-> Was dead : 26

Query ? 4 1 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 47
-> Has recovered : 13
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 20
Quantity: 13 people

Query ? 4 4 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 303
-> Has recovered : 0
-> Was dead : 26

